documentation/features/cpuset.md

```diff
@@ -5,51 +5,48 @@ From Kernel documentation:
 _" Cpusets provide a mechanism for assigning a set of CPUs and Memory Nodes to
 a set of tasks."_
 
-The Kata agent brings compatibility the cgroup cpuset CPU on the guest side.
+The Kata agent brings compatibility to the cgroup cpuset CPU on the guest side.
 
 The cpuset CPU cgroup will be applied on two events:
 
-- containers creation.
+- containers creation
 
-- container update.
+- container update
 
-When the runtime request to apply cpuset cgroup to the agent, the amout of
-vcpus available may be not the same to the required in the request.
+When the runtime requests to apply cpuset cgroup to the agent, the amount of
+vcpus available might not be the same to the required vcpus in the request.
 
-This is becasue the request from the agent client (the Kata runtime) pass
-cpusets that were requested to be placed on the host. This is to isolate the
-container workload on some specific host CPUs. The runtime pass the
-requested cpuset to the agent, that will try to apply the cgroup cpuset on the
-guest.
+This is because the request from the agent client (i.e. the Kata runtime)
+passes cpusets that are requested to be placed on the host. This isolates the
+container workload on some specific host CPUs. The runtime passes the requested
+cpuset to the agent, which tries to apply the cgroup cpuset on the guest.
 
-Because the runtime will only hot-plug and amount the CPUs calculated based in
-the container period and quota, the VM wont have the same amount CPUs than the host.
-guest.
+The runtime only calculates and hot-plugs the CPUSs based on the container
+period and quota. This is why the VM will not have the same amount of CPUs as
+the host.
 
 Example:
 
 docker run -ti --cpus 2 --cpuset 0,1 busybox
  
-The expected result is that the container is limited to the time of 2 CPUs, but
-is only allowed to be scheduled on CPUs 0 and 1 
+This should result with the container limited to the time of 2 CPUs, but is
+only allowed to be scheduled on CPUs 0 and 1.
 
-In a similar case and a valid traditional container is the following:
+The following is an example of a similar case with a valid traditional container:
 
 docker run -ti --cpus 2 --cpuset 2,3,4 busybox
 
-Where the container is limited to 2 CPUs and can be scheduled on CPU 2,3,4.
+Here, the container is limited to 2 CPUs and can be scheduled on CPU 2, 3, and
+4.
 
-In the case of the Kata. The Kata runtime will only hotplug 2 cpus, making
-imposible to request to the guest kernel to schedule the workload on vcpu
-3,4.
+The Kata runtime only hotplugs 2 CPUs, making it impossible to request that the
+guest kernel schedules the workload on vcpu 3 and 4.
 
 ## cpuset best effort application.
 
-Kata will try agent will evaluate this request and see if is possible to 
-apply the cpuset request on the guest.
+The Kata agent evaluates the request to see if it is possible to apply the
+cpuset request onto the guest.
```

Comment:
Can't you fix this directly on the first commit introducing this new documentation?